cmd/env/main.go

```diff
@@ -9,6 +9,6 @@ import (
 func main() {
 	for _, e := range os.Environ() {
 		pair := strings.SplitN(e, "=", 2)
-		fmt.Println(pair[0], ":", pair[1])
+		fmt.Println(pair[0], "=", pair[1])
```

Comment:
e is already a key-value pair separated by =, so why splitting it then concatenating them again, no need for that